types/ssh2/index.d.ts

```diff
@@ -1364,6 +1364,10 @@ export interface Stats extends Attributes {
 export interface FileEntry {
     filename: string;
     longname: string;
+    attrs: Attributes;
+}
+
+export interface FileEntryWithStats extends FileEntry {
```

Comment:
Maybe we shoud omit `attrs` to show that it is intentionally replaced? This could also prevent potential issues in the future.

```suggestion
export interface FileEntryWithStats extends Omit<FileEntry, 'attrs'> {
```

Reply:
Good call!  Updated.